docs/getting-started/install-endpoint.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[install-endpoint]]
-= Configure and install the {endpoint-cloud-sec} integration
+= Configure and install the {elastic-defend} integration
```

Comment:
IIRC, you said the screenshots in this topic will be updated at a later point once integration onboarding efforts concluded?

Reply:
Correct - the screenshots on this page will be updated as part of https://github.com/elastic/security-docs/issues/2457.